media-gfx/digikam/digikam-9999.ebuild

```diff
@@ -66,7 +67,7 @@ COMMON_DEPEND="
 	media-libs/liblqr
 	>=media-libs/libpgf-6.12.27
 	media-libs/libpng:0=
-	media-libs/opencv:=
+	|| ( <media-libs/opencv-3.0.0:= >=media-libs/opencv-3.1.0:= )
```

Comment:
`:=` is not allowed within `|| ( )`: https://bugs.gentoo.org/show_bug.cgi?id=586238

Use e.g.:
```
media-libs/opencv:=
|| ( >=media-libs/opencv-3.1.0 <media-libs/opencv-3.0.0 )
```